cmd/sync.go

```diff
@@ -31,8 +31,8 @@ to get Kong's state in sync with the input state.`,
 
 func init() {
 	rootCmd.AddCommand(syncCmd)
-	syncCmd.Flags().StringVarP(&syncCmdKongStateFile,
-		"state", "s", "kong.yaml", "file containing Kong's configuration. "+
+	syncCmd.Flags().StringSliceVarP(&syncCmdKongStateFile,
+		"state", "s", []string{"kong.yaml"}, "file containing Kong's configuration. "+
```

Comment:
Let's change the help test to instead:
```
file(s) containing Kong's configuration.
This flag can be specified multiple times for multiple files.
Use '-' to read from stdin.
```

Please note the line jumps. We want that in the `--help` output. 
Please change this in all other commands as well.